patches/third_party-blink-renderer-bindings-core-v8-to_v8_traits.h.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/third_party/blink/renderer/bindings/core/v8/to_v8_traits.h b/third_party/blink/renderer/bindings/core/v8/to_v8_traits.h
+index 060b0f1183a4e4d32dadb7cf607266d0355c2643..ee1ff46a855a3cbf0de69e6221187ac63e67f7c5 100644
+--- a/third_party/blink/renderer/bindings/core/v8/to_v8_traits.h
++++ b/third_party/blink/renderer/bindings/core/v8/to_v8_traits.h
+@@ -722,6 +722,15 @@ struct ToV8Traits<IDLNullable<IDLSequence<T>>> {
+       return v8::Null(script_state->GetIsolate());
+     return ToV8Traits<IDLSequence<T>>::ToV8(script_state, *value);
+   }
++
++  template <typename ContainerType>
++  [[nodiscard]] static v8::Local<v8::Value> ToV8(
++      ScriptState* script_state,
++      const ContainerType* value) {
++    if (!value)
++      return v8::Null(script_state->GetIsolate());
++    return ToV8Traits<IDLSequence<T>>::ToV8(script_state, *value);
++  }
```

Comment:
this patch is required for PageGraph. We mark all dictionaries and unions with `OUTPUT` when PG is enabled here:
https://github.com/brave/brave-core/blob/afb81cec78bb318ba2d4d6902f8d3f4435fd313c/chromium_src/third_party/blink/renderer/bindings/scripts/web_idl/idl_compiler.py#L19-L27

Reply:
Fixed in 95416c4125f2ae7044740eaba49d05fba9d95944